overlord/hookstate/hookmgr.go

```diff
@@ -107,10 +112,11 @@ func Manager(s *state.State) (*HookManager, error) {
 	})
 
 	manager := &HookManager{
-		state:      s,
-		runner:     runner,
-		repository: newRepository(),
-		contexts:   make(map[string]*Context),
+		state:       s,
+		runner:      runner,
+		repository:  newRepository(),
+		contexts:    make(map[string]*Context),
+		hijackedMap: make(map[hijackKey]hijackFunc),
```

Comment:
should this be hijackMap now?

Reply:
I was thinking at this point its hijacked already, but it seems `hijackMap` is cleaner and more consistent with hijack{Key,Func} so :+1: - I fix it :)